_posts/2016-08-20-letsencrypt.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: "Moving To Let's Encrypt TLS Certificate"
```

Comment:
Yep, you got it right.  I'll see if I can embed quotes in the title (should be able to...hopefully...)